tests/h/search/index_test.py

```diff
@@ -9,6 +9,21 @@
 from h.search import index
 
 
+class TestIndex(object):
+    def test_index(self, factories, pyramid_request, es_client):
+        annotation = factories.Annotation.build(id="test_annotation_id")
+
+        index.index(es_client, annotation, pyramid_request)
+
+        es_client.conn.indices.refresh(index=es_client.index)
```

Comment:
I wanna rewrite this test to be an actual more useful test. But the refresh is to synchronously refresh the Es index, so that the search query immediately below sees the annotation that was just indexed. Otherwise it happens asynchronously.

This probably shouldn't be done in each individual test though, should maybe find a way, maybe in the test fixtures, to do it automatically.